README.md

```diff
@@ -12,7 +12,9 @@ See the [Frequently Asked Questions](#frequently-asked-questions) to learn more
 Requirements
 ------------
 
-- iOS 9.0+
+- iOS 14.0+
+- Xcode 14.2+
+- Swift 5.7+, or Objective-C
```

Comment:
In the CHANGELOG we say `Xcode 14.3+ and Swift 5.8+` and here we have `Xcode 14.2+ and Swift 5.7+`. We should likely align the two.

Reply:
Ah yes, I wanted to require 14.3+, but that requires a CI update. Will leave that for a separate PR. Will also update these to match.